### PR TITLE
doc: Add link to Visual Studio build readme

### PR DESCRIPTION
**Pull request by maitrebitcoin**
I have searched how to do it in this doc for some time :-(
I think it might help other newbies interested in building with visual studio.

### CODE REVIEW

File: doc/build-windows.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ Build using:
     CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/
     make
 
+## Building using a native compiler tool chain such as Visual Studio
+
+Full instructions are available in [README.md](..//build_msvc/README.md).
```
Comment (MarcoFalke) [inline doc/build-windows.md:106]: This is already mentioned in the bullet point in the first section

> On Windows, using a native compiler tool chain such as Visual Studio.

Please just add the link there

Comment (maitrebitcoin) [reply doc/build-windows.md]: Yes this is better.
new commit added

File: doc/build-windows.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The options known to work for building Bitcoin Core on Windows are:
 and is the platform used to build the Bitcoin Core Windows release binaries.
 * On Windows, using [Windows
 Subsystem for Linux (WSL)](https://docs.microsoft.com/windows/wsl/about) and the Mingw-w64 cross compiler tool chain.
-* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com).
+* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com). See [README.md](../build_msvc/README.md).
```
Comment (MarcoFalke) [inline doc/build-windows.md:12]:
```suggestion
* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com). See [README.md](/build_msvc/README.md).
```

the link can be absolute

**Review by maflcko — COMMENTED** [review]
Please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Review by hebasto — COMMENTED** [review]
@maitrebitcoin 
Welcome and thank you for your contribution!

Tested a96fc40f2dc0aa9e319fbe4addee4a9240df805b, clicked the link and it works.

Mind [squashing](https://github.com/bitcoin/bitcoin/pull/19061#pullrequestreview-417395780) three commits into the one?
> Please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by maitrebitcoin** [comment]
I tried to squash, but I'm  not sure this is all OK.

**Comment by fanquake** [comment]
Thanks, however this still needs fixing up. When you are done squashing there should only be a single commit in this PR.

**Comment by MarcoFalke** [comment]
Try something like this

```
git reset --soft core/master
git commit -m 'doc : add link to readme.md in the first section'

**Comment by maitrebitcoin** [comment]
Looks like i managed to do it somehow.
Thanks for the tips and your patience.

**Review by hebasto — APPROVED** [review]
ACK 1c91ffefcf54d22f805c86318ea32d23b9d20c96, a new link works as expected :)

**Pull request merged**